packages/commonwealth/client/scripts/views/pages/CommunityManagement/AdminsAndModerators/AdminsAndModerators.scss

```diff
@@ -0,0 +1,35 @@
+@import '../../../../../styles/shared';
+
+.AdminsAndModerators {
+  display: grid;
+  grid-template-columns: 3fr 1fr;
+  gap: 69px;
+  padding: 22px 60px;
+  padding-bottom: 100px;
```

Comment:
let's make sure that implemenation for the layout with sidebar is the same for all admin pages